MakeDetailViewController.m

```diff
@@ -0,0 +1,102 @@
+//
+//  MakeDetailViewController.m
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/10/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import "MakeDetailViewController.h"
+#import "Masonry.h"
+
+
+@interface MakeDetailViewController ()
+
+@end
+
+@implementation MakeDetailViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    UIImageView *backgroundImage = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"tires"]];
+    [backgroundImage setFrame:self.view.frame];
+    [self.view addSubview:backgroundImage];
+    [self.view sendSubviewToBack:backgroundImage];
+    backgroundImage.contentMode = UIViewContentModeScaleToFill;
+    
+    self.makeNameArabicString = [[NSString alloc]initWithString:self.make.nameAr];
+    self.makeNameEnglishString = [[NSString alloc]initWithString:self.make.nameEn];
+    self.makeImageViewLargeURLString = [[NSString alloc]initWithString:self.make.imageLargeURL];
+    
+    // Do any additional setup after loading the view.
+    self.view.backgroundColor = [UIColor whiteColor];
+    
+    self.makeImageViewLarge = [[UIImageView alloc]initWithFrame:self.view.frame];
+    self.makeImageViewLarge.contentMode = UIViewContentModeScaleAspectFit;
+    [self.view addSubview:self.makeImageViewLarge];
+    
+    self.makeImageViewLarge.image = [UIImage imageWithData:[NSData dataWithContentsOfURL:[NSURL URLWithString:self.makeImageViewLargeURLString]]];
+    
+    
+    [self.makeImageViewLarge mas_makeConstraints:^(MASConstraintMaker *make) {
+        make.bottom.equalTo(self.view.mas_bottom).with.offset(-10);
+        make.leading.equalTo(self.view.mas_leading).with.offset(10);
+        make.trailing.equalTo(self.view.mas_trailing).with.offset(-10);
+        make.top.equalTo(self.view.mas_top).with.offset(10);
+    }];
+
+    self.makeNameEnglish = [[UILabel alloc]init];
+    [self.makeNameEnglish sizeToFit];
+    self.makeNameEnglish.backgroundColor = [UIColor colorWithWhite:0 alpha:0.5];
+    [self.makeNameEnglish setTextAlignment:NSTextAlignmentCenter];
+    [self.makeNameEnglish setTextColor:[UIColor whiteColor]];
+    [self.makeNameEnglish setFont:[UIFont boldSystemFontOfSize:25]];
+    [self.makeNameEnglish setText:self.makeNameEnglishString];
+    [self.view addSubview:self.makeNameEnglish];
```

Comment:
- I don't know what is makeNameEnglish you should make its name makeNameEnglishLabel 
- there's no need to create makeNameEnglishString you can direct set the make arabic name into the label text 
- no need to use the method setText you can use the property text directly
it should be like self.makeNameEnglishLabel.text = self.make.nameEn